datasource/models/banktrack.py

```diff
@@ -97,3 +96,24 @@ def _generate_tag(cls, bt_tag, increment=0, existing_tags=None):
             return bt_tag
         else:
             return cls._generate_tag(og_tag, increment=increment + 1, existing_tags=existing_tags)
+
```

Comment:
No action required on this one, but FYI: The Banktrack API is giving us much, much richer information than we are currently saving. It may be worth saving that in the future.

for example:
- What projects a bank is funding, along with their years, categorical information, amount(s) and a human-written description

Reply:
see above Wikidata: happy to do that, but I think better on a future PR